hosted_control_planes/hcp-disconnected/hcp-deploy-dc-virt.adoc

```diff
@@ -0,0 +1,68 @@
+
+:FeatureName: {hcp-capital} on {VirtProductName} in a disconnected environment
+include::snippets/technology-preview.adoc[]
+
+[id="prerequisites_{context}"]
+== Prerequisites
+
+* You have a disconnected {product-title} environment serving as your management cluster.
+* You have an internal registry to mirror images on. For more information, see xref:../../installing/disconnected_install/index.adoc[Disconnected installation mirroring].
+
+include::modules/hcp-dc-image-mirror.adoc[leveloffset=+1]
+include::modules/hcp-dc-apply-objects.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* For more information about mirroring nightly or CI versions of {product-title}, see xref:../../installing/disconnected_install/installing-mirroring-disconnected.adoc#installing-mirroring-disconnected[Mirroring images for a disconnected installation using the oc-mirror plugin].
+
+[id="hcp-dc-mce-virt"]
+== Deploying {mce-short} for a disconnected installation of {hcp}
+
+The {mce} plays a crucial role in deploying clusters across providers. If you do not have {mce-short} installed, review the following documentation to understand the prerequisites and steps to install it:
+
+* link:https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.11/html/clusters/cluster_mce_overview#mce-intro[About cluster lifecycle with multicluster engine operator]
+* link:https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.11/html/clusters/cluster_mce_overview#mce-install-intro[Installing and upgrading multicluster engine operator]
+
+[id="hcp-dc-tls-virt"]
+== Configuring TLS certificates for a disconnected installation of {hcp}
+
+To ensure proper function in a disconnected deployment, you need to configure the registry CA certificates in the management cluster and the worker nodes for the hosted cluster.
+
+include::modules/hcp-dc-tls-mgmt.adoc[leveloffset=+2]
+include::modules/hcp-dc-tls-hosted.adoc[leveloffset=+2]
+
+[id="hcp-dc-virt-hosted"]
+== Creating a hosted cluster on {VirtProductName}
+
+A hosted cluster is an {product-title} cluster with its control plane and API endpoint hosted on a management cluster. The hosted cluster includes the control plane and its corresponding data plane.
+
+include::modules/hcp-virt-reqs.adoc[leveloffset=+2]
+include::modules/hcp-virt-create-hc-cli.adoc[leveloffset=+2]
+include::modules/hcp-virt-ingress-dns.adoc[leveloffset=+2]
+
+[id="hcp-dc-virt-ingress-dns-custom"]
+=== Customizing ingress and DNS behavior
+
+If you do not want to use the default ingress and DNS behavior, you can configure a KubeVirt hosted cluster with a unique base domain at creation time. This option requires manual configuration steps during creation and involves three main steps: cluster creation, load balancer creation, and wildcard DNS configuration.
```

Comment:
Probably optional but these steps may use bullets.
"....involves three main steps: cluster creation, load balancer creation, and wildcard DNS configuration."